weldx/transformations/local_cs.py

```diff
@@ -360,6 +364,7 @@ def _check_and_normalize_orientation(orientation: xr.DataArray) -> xr.DataArray:
             orientation,
             input_core_dims=[["c"]],
             output_core_dims=[["c"]],
+            keep_attrs=True,
```

Comment:
I am not sure how or if default arguments changed with regards to keeping xarray attributes, however this is what we want

Reply:
Eventually, we should make these arguments explicit to avoid breaking changes due to upstream default argument changes?